roaringbitmap/src/main/java/org/roaringbitmap/buffer/MappeableBitmapContainer.java

```diff
@@ -1269,27 +1269,26 @@ public int nextSetBit(final int i) {
   }
 
   /**
-   * Find the index of the next unset bit greater or equal to i, returns -1 if none found.
+   * Find the index of the next clear bit greater or equal to i.
    *
    * @param i starting index
-   * @return index of the next unset bit
+   * @return index of the next clear bit
    */
-  public short nextUnsetBit(final int i) {
-    int x = i / 64;
+  public int nextClearBit(final int i) {
+    int x = i >> 6; // i / 64 with sign extension
```

Comment:
I would prefer we don't put comments like this in... It's similar to the famous one below :)

```java
// increment i
++i;
```

Reply:
True, a similar comment is present in `nextSetBit`. I can remove both.